home/views.py

```diff
@@ -265,47 +254,49 @@ def updated_avatar(request, user_id):
 
     return HttpResponse(hacker.avatar_url)
 
-@login_required
+
 def feed(request):
-    ''' Atom feed of all new posts. '''
+    """Atom feed of all new posts."""
 
-    postList = list(Post.objects.all().order_by('-date_updated'))
+    token = request.GET.get('token')
+    if not token or authenticate(token=token) is None:
+        raise Http404
 
-    for post in postList:
-        user = User.objects.get(blog__id__exact=post.blog_id)
-        post.author = user.first_name + " " + user.last_name
+    return LatestEntriesFeed()(request)
 
-    context = Context({
-        "postList": postList,
-        "domain": settings.SITE_URL
-    })
 
-    return render(request, 'home/atom.xml', context, content_type="text/xml")
+@login_required
+def refresh_token(request):
+    """Refresh a users' auth token."""
 
+    hacker = Hacker.objects.get(user=request.user)
+    hacker.token = get_random_unique_token()
+    hacker.save()
 
-@login_required
-def item(request, slug):
+    profile_url = reverse('profile', kwargs={'user_id': request.user.id})
+    return HttpResponseRedirect(profile_url)
 
-    if request.method == 'POST':
-        if request.POST['content']:
-            Comment.objects.create(
-                slug=generate_random_id(),
-                user=request.user,
-                post=Post.objects.get(slug=slug),
-                parent=None,
-                date_modified=timezone.now(),
-                content=request.POST['content'],
-            )
+def get_random_unique_token():
+    """Get a random token after ensuring that it is unique.
 
-    post = get_post_info(slug)
+    The model is passed as an argument to allow using this function in data
+    migrations.
```

Comment:
This is an outdated docstring. The code has been duplicated in the migration to allow backward/forward migrations without depending on this function to exist in the app's code.